### PR TITLE
Improve fetch's move base by fetchrobotics's default params

### PR DESCRIPTION
**Pull request by 708yamaguchi**
~Please merge after https://github.com/jsk-ros-pkg/jsk_robot/pull/1455~

This PR includes https://github.com/knorth55/jsk_robot/pull/124, https://github.com/knorth55/jsk_robot/pull/125 and https://github.com/knorth55/jsk_robot/pull/126

We thought `TebLocalPlanner` is the best for fetch15 and fetch1075, so we created https://github.com/jsk-ros-pkg/jsk_robot/pull/1455.

However, we found that the fetchrobtics's default parameter is better for fetch1075.

In this PR, we use the original paramter for fetch15 and use the default parameter for fetch1075.

We tested the parameter with `go-to-kitchen` demo.

### CODE REVIEW

**Comment by sktometometo** [comment]
@708yamaguchi このPRで議論することじゃないかもしれないけど、 relayして出している `/move_base/navigation_plan_viz` と `/move_base/global_plan_viz` は分離する意味が無い気がします。

File: jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch [hunk 1]
```diff
@@ -192,6 +193,25 @@
       <arg name="launch_map_server" value="false" />
       <arg name="odom_topic" value="/odom_combined" />
     </include>
+    <!-- relay plan path topic for visualization -->
```
Comment (sktometometo) [inline jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch:196]: `/move_base/<global planner>/plan` and `/move_base/<local planner>/global_plan` looks almost same except replanning of global plan. What is the purpose of splitting their visulization?
cf. https://wiki.ros.org/base_local_planner#API_Stability

**Comment by k-okada** [comment]
`However, we found that the fetchrobtics's default parameter is better for fetch1075.` That's good lesson! That's always good to imagine what  is the fetchrobotics's engineer's job and how should we cooperate with them.

File: jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch (hunk 1 above)
Comment (708yamaguchi) [reply jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch:196]: @sktometometo 

> looks almost same

Yes, I think so, too.

The only reason I publish both `/move_base/<global planner>/plan` and `/move_base/<local planner>/global_plan` is that they have been visible in RViz config in fetch for a long time.

I will create another PR to reduce visualization.
Which do you think should be visualized, `/move_base/<global planner>/plan` or `/move_base/<local planner>/global_plan`?

**Comment by 708yamaguchi** [comment]
I rebased `origin/master` to resolve conflicts.

I will add some commits to reflect the review https://github.com/jsk-ros-pkg/jsk_robot/pull/1456#discussion_r854655008

**Review by sktometometo — APPROVED** [review]

**Pull request merged**